src/index.ts

```diff
@@ -113,14 +126,22 @@ function getPreferredResponseFormat(req: Request): ResponseFormat {
     return 'text';
 }
 
-function sendYouchError(req: Request, res: Response, err: NormalizedException): Promise<void> {
+function sendYouchError(
+  req: Request,
+  res: Response,
+  err: NormalizedException,
+  options?: ErrorReporterOptions
+): Promise<void> {
     const youch = new Youch(err, req);
-    return youch.toHTML()
-        .then((html: string) => {
-            res.writeHead(err.statusCode, {'content-type': 'text/html'});
-            res.write(html);
-            res.end();
-        });
+    const asHtml =
```

Comment:
Here, you could let `youch` be a non-constant variable (eg. `let youchBuilder = new Youch(...`). Then loop over (or reduce) `options.links` to  `addLink` on `youchBuilder` (I am assuming addLink is meant to be called multiple times to add multiple links).

Reply:
Fixed. Added a reduce, seemed more readable.